### DISCUSSION TITLE
Help: How can I run less commnd?

### DISCUSSION THREAD

**Discussion by Nadim147c** [google/zx#862] [Help needed]
I'm trying to run the `less` command from js but following code isn't working 
```js
await $({ input: content })`less ${"+" + lineNumber.trim()}`
```
I also tried it with `verbose` but not working. So, is there anyway to run the `less` command?

**Comment by antonmedv** [comment]
In order to run less command you need a tty or a pty. zx does not provide those. 

Can use you `cat` instead?

Or consider something like node-pty.

**Comment by Nadim147c** [comment]
Hey, thanks for letting me know. 

I was initially using cat (`tail +n`). But I needed to preview the previous line, which can't be done with cat or tail command. 

I was trying `node-pty` to work with less command but failed. I know it's outside `zx`, But I would appreciate if you could share any snippet or something.

**Comment by antonmedv** [comment]
> But I needed to preview the previous line


use js

**Comment by Nadim147c** [comment]
I couldn't find any NPM package that helps me to do that. I guess I have to make my own.

Thanks anyway.
